firmware/firmware.ino

```diff
@@ -385,11 +392,33 @@ void loop()
   // Copy incoming message to debug console.
   Serial.print("<- ");
   Serial.println(input_buffer);
-    
+  
   // Attempt to parse command
   handleCommand(input_buffer);
 }
 
+void batteryUpdateLoop()
+{
+  int rawVoltage = analogRead(board::V_BATT);
+  double voltageReading = 0.008879*rawVoltage + 0.09791;
+  //0.03516*rawVoltage+0.05135;
```

Comment:
Unless we know what this is coming from, let's just delete the comment (we can always look up the old function in the git history)